frontend/packages/kubevirt-plugin/src/components/create-vm-wizard/tabs/vm-settings-tab/provision-source.tsx

```diff
@@ -69,18 +75,34 @@ export const ProvisionSourceComponent: React.FC<ProvisionSourceComponentProps> =
     const provisionSourceValue = iGetFieldValue(provisionSourceField);
     const sources = iGet(provisionSourceField, 'sources');
 
+    const [isSourceOpen, setIsSourceOpen] = React.useState(false);
+
+    const onSelect = (e, v) => {
+      onChange(VMSettingsField.PROVISION_SOURCE_TYPE, v.toString());
+      setIsSourceOpen(false);
+    };
+
     return (
-      <FormFieldRow field={provisionSourceField} fieldType={FormFieldType.SELECT}>
+      <FormFieldRow field={provisionSourceField} fieldType={FormFieldType.PF_SELECT}>
         <FormField>
-          <FormSelect onChange={(v) => onChange(VMSettingsField.PROVISION_SOURCE_TYPE, v)}>
-            <FormSelectPlaceholderOption
-              placeholder={getPlaceholder(VMSettingsField.PROVISION_SOURCE_TYPE)}
-              isDisabled={!!provisionSourceValue}
-            />
+          <Select
+            isOpen={isSourceOpen}
+            selections={provisionSourceValue}
+            onToggle={(isOpen) => setIsSourceOpen(isOpen)}
```

Comment:
maybe it would be beneficial to create a single select component that has all the defaults included and stores its own state.
This will come in handy once we include multiple of these in.